html/utilities/keypress.js

```diff
@@ -12,7 +12,7 @@ const isEnterPressed = (e) => e.key === 'Enter' || e.keyCode === 13;
 const isDownPressed = (e) => e.key === 'ArrowDown' || e.keyCode === 40;
 const isLeftPressed = (e) => e.key === 'ArrowLeft' || e.keyCode === 37;
 const isRightPressed = (e) => e.key === 'ArrowRight' || e.keyCode === 39;
-const isUpPressed = (e) => e.key === 'ArrowRight' || e.keyCode === 38;
+const isUpPressed = (e) => e.key === 'ArrowUp' || e.keyCode === 38;
```

Comment:
`key` is deprecated and all our supported browsers use `keyCode`, so I think this typo would only have caused problems in old browsers we don't support.

Reply:
Great catch!